src/clientSideScene/sceneEntities.ts

```diff
@@ -388,14 +407,31 @@ export class SceneEntities {
             sceneObject.object.name === X_AXIS ||
             sceneObject.object.name === Y_AXIS
         )
-        if (!axisIntersection) return
+
+        const arrowHead = getParentGroup(args.intersects[0].object, [ARROWHEAD])
+        const parent = getParentGroup(
+          args.intersects[0].object,
+          SEGMENT_BODIES_PLUS_PROFILE_START
+        )
+        if (
+          !axisIntersection &&
+          !(
+            parent?.userData?.isLastInProfile &&
+            (arrowHead || parent?.name === PROFILE_START)
+          )
+        )
+          return
```

Comment:
We want the same snap "draft point" when the user is close to an arrowhead of the last segment of a profile.